components/landingPage/StudentProjects.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+export default function StudentProjects() {
+  return (
+    <div className="flex flex-col items-center">
+      <h1 className="text-blue-800 text-center text-3xl mb-20 font-bold">
+        Student Projects
+      </h1>
+      <div className="grid grid-cols-1 md:grid-cols-3 gap-10">
+        <div className="flex flex-col items-center relative">
+          <div className="bg-zinc-400 h-80 w-72 flex flex-col justify-center items-center rounded-lg shadow-lg">
```

Comment:
I have added shadow-lg but it doesn't look like there's a shadow there. 
![image](https://github.com/skillify-ca/skillify-web/assets/114423326/7e297d60-5f7d-475c-a37d-c3aee6abd9c7)